conda.yml

```diff
@@ -1,10 +1,9 @@
 name: rnaseq-nf
```

Comment:
Is this file needed any more? Each process now has a single conda package specified, and the `Dockerfile-mamba` hardcodes all the package names..

Reply:
Yep it is being used here still:

https://github.com/nextflow-io/rnaseq-nf/blob/88b8ef803a02137750b8c5e063388e8d6a58ac36/docker/Dockerfile#L3

https://github.com/nextflow-io/rnaseq-nf/blob/88b8ef803a02137750b8c5e063388e8d6a58ac36/docker/Makefile#L6

Reply:
Ah ok, might be worth making `Dockerfile-mamba` do the same thing for consistency then..